src/components/Book.js

```diff
@@ -2,34 +2,48 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/booksSlice';
+import '../styles/Book.css';
 
 function Book({ books }) {
   const dispatch = useDispatch();
 
   return (
-    <div>
+    <div className="book-container">
       <ul>
         {books.map((book) => (
           <li key={book.id}>
             <div>
-              <div>
-                <p>{book.category}</p>
-                <h1>{book.title}</h1>
-                <p>
+              <div className="book-left">
+                <p className="category-book">{book.category}</p>
+                <h2 className="title-book">{book.title}</h2>
+                <p className="author-book">
                   {book.author}
                 </p>
-                <div>
-                  <button type="submit">comments</button>
-                  <div className="border" />
-                  <button type="submit" onClick={() => dispatch(deleteBook(book.id))}>Remove</button>
-                  <div className="border" />
-                  <button type="submit">Edit</button>
-                </div>
+                <ul className="features">
+                  <li type="submit">Comments</li>
+                  <li>
+                    <button className="remove-button" type="submit" onClick={() => dispatch(deleteBook(book.id))}>Remove</button>
+                  </li>
```

Comment:
- [ ] Kindly use the correct font family for the remove button link.
![image](https://github.com/AbuTalha3/bookstore-app/assets/81932478/f5c2709d-6c82-4965-8252-4a98c2901e89)